templates/counter/counter.azurefilesvolume.linux.json

```diff
@@ -35,6 +35,47 @@
     }
   },
   "resources": [
+    {
+      "apiVersion": "2018-09-01-preview",
+      "name": "counterAppNetwork",
+      "type": "Microsoft.ServiceFabricMesh/networks",
+      "location": "[parameters('location')]",
+      "dependsOn": [],
+      "properties": {
+        "kind": "Local",
+        "description": "Azure Service Fabric Mesh Counter Application network.",
+        "networkaddressprefix": "2.0.0.0/16"
```

Comment:
can you change all the network subnets to 10.0.0.0/24?

Reply:
Done